packages/@o3r/core/schematics/index.it.spec.ts

```diff
@@ -43,10 +44,11 @@ function setupLocalRegistry() {
   beforeAll(() => {
     containerId = execSync(`docker run -d -it --rm --name verdaccio -p 4873:4873 -v ${verdaccioFolder}:/verdaccio/conf verdaccio/verdaccio`, {cwd: currentFolder, stdio: 'pipe'}).toString();
     execSync(`echo registry=${registry} > .npmrc`, {cwd: verdaccioFolder, stdio: 'inherit'});
-    execSync('yarn set:version 8.0.0 --include !**/!(dist)/package.json --include !package.json', {cwd: currentFolder, stdio: 'inherit'});
+    execSync('yarn set:version 8.0.0 --include "!**/!(dist)/package.json" --include !package.json', {cwd: currentFolder, stdio: 'inherit', env: {...process.env}});
```

Comment:
`env: {...process.env}` is mandatory? you can not pass the ref with `env: process.env` ?

Reply:
updated